### DISCUSSION TITLE
GitHub Repo Pull

### DISCUSSION THREAD

**Discussion by KolinCunningham** [thebaselab/codeapp#1056] [Q&A]
So Guys I have the git hub app and the code app, I can't figure out how to pull from my repos and save on a local file from their to use the code app. Could someone give me like a 5 step method. I have been banging my head for the past 2 hours on this one. I thought it would be easy.

**Comment by bummoblizard** [comment]
You might be looking for 'clone' instead. This article provides the steps you need to clone a repository: https://code.thebaselab.com/guides/version-control.

**Comment by bummoblizard** [comment]
https://github.com/thebaselab/codeapp/assets/38398443/099ef74f-982a-4fc0-8743-7f8e69745391
